Feat:  Add Message and Link for ToastBar #DS-1213 (#1426)
lmc-eu/spirit-design-system: packages/web-react/src/components/Toast/README.md

Reviewer: Please make sure nowhere else is the function `show` called with a simple string as a message. We need to pass there the object. And if there are docs for the function, we need to update them too.

```

```jsx

> import ToastBarMessage from './ToastBarMessage';

Reviewer 2: Ok, let's discuss this tomorrow or maybe @crishpeen will write some note to this. My point is that the developer who will use this example will not have this relative path in the filesystem. I have also searched the READMEs and there is no relative import. So I wonder what the comment was about 🤔

> import ToastBarMessage from './ToastBarMessage';

Author: I'm sorry my mistake. I misunderstood your comment. The code has been modified.

| `success`     | `check-plain` |
| `warning`     | `warning`     |

### ToastBar Components

The content of `ToastBar` can be assembled from the following subcomponents:

#### ToastBarMessage

`ToastBarMessage` is a subcomponent designated for the main message in `ToastBar`.

Usage example:

```jsx
<ToastBar id="my-toast" isOpen={isOpen} onClose={() => setIsOpen(false)} isDismissible>
  <ToastBarMessage>This is the main toast message.</ToastBarMessage>
</ToastBar>
```

#### API

| Name       | Type        | Default | Required | Description                    |
| ---------- | ----------- | ------- | -------- | ------------------------------ |

> | `children` | `ReactNode` | —       | ✓        | Content of the ToastBarMessage |

Reviewer: Please copypaste the section about escape hatches below. You can take it anywhere else.

Reviewer: This should be everywhere (in twig and react)

Author: I'm not sure if I understand correctly. I added this part to web-twig README.

Reviewer 2: This was meant in @crishpeen comment:
```
On top of the API options, the components accept [additional attributes][readme-additional-attributes].
If you need more control over the styling of a component, you can use [style props][readme-style-props]
and [escape hatches][readme-escape-hatches].
```
This paragraph says that the component is using `UNSAFE_` style props. E.g. we do not want to repeat them in every API table.

| `success`     | `check-plain` |
| `warning`     | `warning`     |

### ToastBar Components

> The content of `Toast Bar` can be assembled from the following components:

Reviewer: ```suggestion
The content of `ToastBar` can be assembled from the following components:
```

> #### ToastBarMessage

Reviewer: I think we also add an API section to subcomponents that don't have API, because even then, they have escape hatches.

| `success`     | `check-plain` |
| `warning`     | `warning`     |

### ToastBar Components

The content of `ToastBar` can be assembled from the following components:

#### ToastBarMessage

`ToastBarMessage` is a component designates for main message in `ToastBar`.

Usage example:

```jsx
<ToastBar id="my-toast" isOpen={isOpen} onClose={() => setIsOpen(false)} isDismissible>
  <ToastBarMessage>This is the main toast message.</ToastBarMessage>
</ToastBar>
```

#### API

| Name       | Type        | Default | Required | Description                    |
| ---------- | ----------- | ------- | -------- | ------------------------------ |

> | `children` | `ReactNode` | —       | ✔        | Content of the ToastBarMessage |

Reviewer 2: ```suggestion
| `children` | `ReactNode` | —       | ✓        | Content of the ToastBarMessage |
```

Please use the "check mark" instead of the "heavy check mark". See https://github.com/lmc-eu/spirit-design-system/pull/1436.

Same in other API tables in this PR.

For example:

```jsx

> import ToastBarMessage from './ToastBarMessage';

Reviewer 2: Please use absolute import from the package.

## Full Example

```jsx
import {Button, Toast, ToastBar} from '@lmc-eu/spirit-web-react/components';
import ToastBarMessage from "./ToastBarMessage";

> import ToastBarLink from "./ToastBarLink";

Reviewer: everything is gonna be imported from our package, not locally

## Full Example

```jsx

> import {Button, Toast, ToastBar} from '@lmc-eu/spirit-web-react/components';

Reviewer 2: ```suggestion
import { Button, Toast, ToastBar } from '@lmc-eu/spirit-web-react/components';
```
